spec/proceduralSpec.php

```diff
@@ -0,0 +1,13 @@
+<?php
+namespace procedural\spec;
+
+use function prodecural\echoScript;
+
+describe('procedural', function () {
+  describe('->echoScript()', function () {
+    it('echos input string inside a script tag', function () {
```

Comment:
we use `php:^5.5|^7.0` , see travis build https://travis-ci.org/sitrunlab/learn-kahlan/builds/234091631 so it still required to use this check